test/unit/math/gpu/diagonal_multiply_test.cpp

```diff
@@ -5,12 +5,8 @@
 #include <algorithm>
 #ifdef STAN_OPENCL
 TEST(MathMatrix, multiply_m_exception_pass_diagonal_mul) {
-  stan::math::matrix_d m0;
-  m0.resize(3, 2);
-  m0 << 1, 1, 1, 1, 1, 1;
-  stan::math::matrix_d m1;
-  m1.resize(1, 3);
-  m1 << 1, 1, 1;
+  stan::math::matrix_d m0 = stan::math::matrix_d::Ones(3, 2);
```

Comment:
You can also do `auto m0 = stan::math::matrix_d::Ones(3, 2);` and you could do a `using stan::math::matrix_d;` at the top of a test file if you wanted. No need to change this now.